howtos/how-to-create-a-popup-inbox/js/script.js

```diff
@@ -0,0 +1,115 @@
+(function (t, a, l, k, j, s) {
+  s = a.createElement("script");
+  s.async = 1;
+  s.src = "https://cdn.talkjs.com/talk.js";
+  a.head.appendChild(s);
+  k = t.Promise;
+  t.Talk = {
+    v: 3,
+    ready: {
+      then: function (f) {
+        if (k)
+          return new k(function (r, e) {
+            l.push([f, r, e]);
+          });
+        l.push([f]);
+      },
+      catch: function () {
+        return k && new k();
+      },
+      c: l,
+    },
+  };
+})(window, document, []);
+
+const popupInboxContainer = document.getElementById("popup-inbox-container");
+const talkjsContainer = document.getElementById("talkjs-container");
+const popupTriggerBtn = document.getElementById("popup-trigger-btn");
+
+const appId = "<APP_ID>";
+const currentUser = {
+  id: "1234",
+  name: "me",
+  email: "me@email.com",
+  photoUrl: "https://talkjs.com/images/avatar-1.jpg",
+  welcomeMessage: "What`s up?",
+  role: "default",
+};
+
+const conversationItems = [
```

Comment:
I think it would be better to have a conversation with only one other user. The other conversations won't appear in the chat history anyway now there's no message sent to them, and we don't need multiple conversations to demonstrate the main point of the tutorial.